.github/new-prs-labeler.yml

```diff
@@ -479,6 +479,15 @@ llvm:analysis:
   - llvm/lib/Analysis/
   - llvm/include/llvm/Analysis/
 
+llvm:adt:
+  - llvm/include/llvm/ADT/
```

Comment:
I think all these patterns need to end with ** .

Reply:
Is there any way to test this before submitting?

Reply:
Yes, see: https://github.com/llvm/llvm-project/pull/66145#discussion_r1325155972

Reply:
Thanks, I tested the patterns using:
```js
const minimatch = require('minimatch')

function check_pattern(file, pattern) {
    match = minimatch(file, pattern)
    console.log(file + " : " + pattern + " : " + match)
}

check_pattern('llvm/include/llvm/ADT/Foo.cpp', 'llvm/**/ADT/*')
check_pattern('llvm/lib/ADT/Foo.cpp', 'llvm/**/ADT/*')
check_pattern('llvm/lib/unittests/ADT/Foo.cpp', 'llvm/**/ADT/*')
```